universum/modules/artifact_collector.py

```diff
@@ -112,7 +112,10 @@ def create_text_file(self, name):
                     raise CriticalCiException(text)
 
             self.file_list.add(file_name)
-            file_path = self.automation_server.artifact_path(self.artifact_dir, os.path.basename(file_name))
+            try:
+                file_path = self.automation_server.artifact_path(self.artifact_dir, os.path.basename(file_name))
+            except RuntimeError:
+                file_path = file_name
```

Comment:
How does this change look when encountered in actual execution?

Reply:
We cannot provide a link to an individual file in the archive, or to the archive in general, without first creating it (which now looks like unnecessary functionality). therefore, instead of a file reference, we return file path on CI:
```
1.1. Disconnecting
==> Review can be found here: https://swarm.net/reviews/12345/
2. Preparing repository
  ==> Adding file /__w/project/project/artifacts/REPOSITORY_STATE.txt to artifacts...
```